src/mapper/github.mapping.ts

```diff
@@ -17,9 +17,9 @@ export function mapProject(
 }
 
 export function mapProjects(
-  repositories: any[],
-  repositoriesLanguages: any[],
-  repositoriesContributors: any[]
+  repositories: any,
```

Comment:
can you change repository to an array.

The others are objects but repositories is an array.

Reply:
I made the required change.